bridge/eth/client.go

```diff
@@ -244,6 +256,12 @@ func (c *Client) SignersCount(context.Context, bridge.ActionId) uint {
 	return uint(len(c.broadcaster.Signatures()))
 }
 
+// QuorumProvider
+
+func (c *Client) GetQuorum(ctx context.Context) (*big.Int, error) {
+	return c.bridgeContract.Quorum(&bind.CallOpts{Context: ctx})
```

Comment:
Quorum as a big Int ? this should be a small value.

Reply:
It might be true, but I take the value from the eth contract, this needs to be changed in the contract, please open a issue for it.

Reply:
You get as BigInt. but it is completely normal to convert uint64 and add sanity checks. Please do it!